### PR TITLE
Add missing 'container cp' alias and document missing 'container update' command

### PR DESCRIPTION
**Pull request by josegonzalez**
Some notes

- Also reorder the missing update command to better match the container update command (it is in the same management namespace)
- 'docker cp' is an alias for 'docker container cp', and podman should have the equivalent alias.

### CODE REVIEW

File: cmd/podman/container.go [hunk 1]
```diff
@@ -54,6 +54,7 @@ var (
 		_checkpointCommand,
 		_containerExistsCommand,
 		_contInspectSubCommand,
+		_cpCommand,
```
Comment (TomSweeneyRedHat) [inline cmd/podman/container.go]: @baude does this need to be added to mainCommands in cmd/podman/main.go too?

**Comment by rhatdan** [comment]
@josegonzalez Thanks for the PR.

Could you also update the man page docs/podman-container.1.md to add podman container cp.
Also update completions for bash and zsh to include it.

**Comment by mheon** [comment]
The Gating failure is our manpage checker - updating `podman-container.1.md` as @rhatdan requested should resolve it

File: cmd/podman/container.go (hunk 1 above)
Comment (josegonzalez) [reply cmd/podman/container.go]: This confused me a bit, since the command is documented and there, but apparently not in the main?

**Comment by josegonzalez** [comment]
@rhatdan I don't see how to update the zsh completions, not sure there is anything to do there.

**Comment by mheon** [comment]
ZSH is autogenerated, you should be fine

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**